src/index.jsx

```diff
@@ -36,7 +36,10 @@ subscribe(APP_READY, () => {
 });
 
 subscribe(APP_INIT_ERROR, (error) => {
-  ReactDOM.render(<ErrorPage message={error.message} />, document.getElementById('root'));
+  ReactDOM.render(
+    <ErrorPage message={error.message} />,
+    document.getElementById('root'),
+  );
 });
```

Comment:
@Kelketek This is unrelated to your code change, but I'm having issues installing this package.

```
morel:demo-mfe$ npm version
{
  '@open-craft/demo-mfe': '0.1.0',
  npm: '8.6.0',
  node: '18.0.0',
  v8: '10.1.124.8-node.13',
  uv: '1.43.0',
  zlib: '1.2.11',
  brotli: '1.0.9',
  ares: '1.18.1',
  modules: '108',
  nghttp2: '1.47.0',
  napi: '8',
  llhttp: '6.0.4',
  openssl: '3.0.2+quic',
  cldr: '41.0',
  icu: '71.1',
  tz: '2022a',
  unicode: '14.0',
  ngtcp2: '0.1.0-DEV',
  nghttp3: '0.1.0-DEV'
}

# install ran fine
morel:demo-mfe$ npm install

# start had issues
morel:demo-mfe$ npm start

> @open-craft/demo-mfe@0.1.0 start
> fedx-scripts webpack-dev-server --progress
<snip>
Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:67:19)
    at Object.createHash (node:crypto:133:10)
    at BulkUpdateDecorator.hashFactory (/home/pomegranited/dev/edx/src/demo-mfe/node_modules/webpack/lib/util/createHash.js:145:18)
    at BulkUpdateDecorator.update (/home/pomegranited/dev/edx/src/demo-mfe/node_modules/webpack/lib/util/createHash.js:46:50)
    at OriginalSource.updateHash (/home/pomegranited/dev/edx/src/demo-mfe/node_modules/webpack/node_modules/webpack-sources/lib/OriginalSource.js:131:8)
    at NormalModule._initBuildHash (/home/pomegranited/dev/edx/src/demo-mfe/node_modules/webpack/lib/NormalModule.js:880:17)
    at handleParseResult (/home/pomegranited/dev/edx/src/demo-mfe/node_modules/webpack/lib/NormalModule.js:946:10)
    at /home/pomegranited/dev/edx/src/demo-mfe/node_modules/webpack/lib/NormalModule.js:1040:4
    at processResult (/home/pomegranited/dev/edx/src/demo-mfe/node_modules/webpack/lib/NormalModule.js:755:11)
    at /home/pomegranited/dev/edx/src/demo-mfe/node_modules/webpack/lib/NormalModule.js:819:5 {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}

# ref https://stackoverflow.com/a/69746937/4302112
morel:demo-mfe$ export NODE_OPTIONS=--openssl-legacy-provider

# then the MFE ran ok
morel:demo-mfe$ npm start
```

But -- visiting in the browser gave me CORS issues:
```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://localhost:18000/login_refresh. (Reason: CORS request did not succeed). Status code: (null).
```

Is there something else I'm supposed to change in the platform config?